src/Document/Element.ts

```diff
@@ -185,6 +187,7 @@ export default abstract class Element {
 			const mask = this.getStyle('mask').getDefinition<MaskElement>();
 
 			if (mask) {
+				this.applyEffects(ctx);
 				mask.apply(ctx, this);
```

Comment:
we can also potentially save context, apply, then load context back if that helps with something